software/controller/lib/hal/buzzer.cpp

```diff
@@ -46,7 +44,7 @@ void HalApi::InitBuzzer() {
   TimerReg *tmr = Timer3Base;
```

Comment:
Maybe? This will probably naturally happen because we may have to be instantiating multiple buzzers for the new PCB, which I am not sure if they are on the same or different timers or what...?

But also, in a lot of cases, you don't even need the temporary `tmr` variable and can work on the global (ugh) register directly. There is also the possibility that we abstract the timer interface some more and make it unnecessary to access the register directly.